src/recipe.cpp

```diff
@@ -609,24 +609,25 @@ struct prof_penalty {
 };
 
 static std::string profstring( const prof_penalty &prof,
-                               std::string &color )
+                               std::string &color,
+                               const std::string &name_color = "cyan" )
 {
     std::string mitigated_str;
     if( prof.mitigated ) {
         mitigated_str = _( " (Mitigated)" );
     }
 
     if( prof.time_mult == 1.0f ) {
-        return string_format( _( "<color_cyan>%s</color> (<color_%s>%gx\u00a0failure</color>%s)" ),
-                              prof.id->name(), color, prof.failure_mult, mitigated_str );
+        return string_format( _( "<color_%s>%s</color> (<color_%s>%gx\u00a0failure</color>%s)" ),
+                              name_color, prof.id->name(), color, prof.failure_mult, mitigated_str );
     } else if( prof.failure_mult == 1.0f ) {
-        return string_format( _( "<color_cyan>%s</color> (<color_%s>%gx\u00a0time</color>%s)" ),
-                              prof.id->name(), color, prof.time_mult, mitigated_str );
+        return string_format( _( "<color_%s>%s</color> (<color_%s>%gx\u00a0time</color>%s)" ),
+                              name_color, prof.id->name(), color, prof.time_mult, mitigated_str );
     }
 
     return string_format(
-               _( "<color_cyan>%s</color> (<color_%s>%gx\u00a0time, %gx\u00a0failure</color>%s)" ),
-               prof.id->name(), color, prof.time_mult, prof.failure_mult, mitigated_str );
+               _( "<color_%s>%s</color> (<color_%s>%gx\u00a0time, %gx\u00a0failure</color>%s)" ),
+               name_color, prof.id->name(), name_color, prof.time_mult, prof.failure_mult, mitigated_str );
```

Comment:
Is it intentional to also change colour of multipliers? I notice that _Principles of Metalworking (1.5x time, 5x failure)_ is no longer in yellow.
<img width="1072" alt="unpatched" src="https://user-images.githubusercontent.com/21075502/103637400-70ca1280-4f19-11eb-8f2f-3fbfd9145eda.png">
(unpatched)

<img width="1072" alt="patched" src="https://user-images.githubusercontent.com/21075502/103637397-70317c00-4f19-11eb-992d-d11a123a6c68.png">
(patched)

Reply:
i ll take a look at it. the goal was to only change the color of the advanced profs you can't learn and not all of them.